### ISSUE TITLE
subjects should have different ML models for prediction of relevancy

### ISSUE THREAD

**Issue by brunoamaral** [brunoamaral/gregory#236]
In this context, Subject refers tp a Research Subject, or a Topic of research. When we create a new Source for articles, we have to assign it to an existing subject that belongs to that User/Team

The ML model Requires a dataset with some articles already flagged.

**Comment by brunoamaral** [comment]
depends on #329

**Comment by brunoamaral** [comment]
duplicate of #340
